lib/tests/streamlit/components_test.py

```diff
@@ -26,123 +28,129 @@
 from streamlit.components import MarshallComponentException
 from streamlit.components import ComponentRegistry
 from streamlit.components import ComponentRequestHandler
-from streamlit.components import register_component
+from streamlit.components import CustomComponent
+from streamlit.components import register_component as register_component
+from streamlit.components import declare_component as declare_component
 from tests.testutil import DeltaGeneratorTestCase
+from streamlit.errors import StreamlitAPIException
+from streamlit.proto.ComponentInstance_pb2 import ArgsDataframe
 
-JAVASCRIPT_1 = """
-return function (args, st) => {
-  return <div>Hello, lovely world!</div>
-}
-"""
 
-JAVASCRIPT_2 = """
-export default (args, st) => {
-  return <span>Goodbye, cruel world!</span>
-}
-"""
+URL = "http://localhost:3001"
+PATH = "component_template/build"
```

Comment:
Can you change these to use obvious test values, to make potential future test failures more obvious? For example:

```
URL = "http://not.a.real.url:3001"
PATH = "not/a/real/path"
```